### PR TITLE
docs(components): Update Packages section

### PR DESCRIPTION
**Pull request by taylorvnoj**
### Added

Packages:
- Components
- Eslint Config
- Hooks
- Stylelint Config

Note: Docz Tools is not being added and Formatters doesn't contain any content

### Changed

Updated the way these `README`'s get loaded in Storybook -> by loading from the original `README` instead of duplicating a file, **except** for these instances:
- _Hooks_: Decided to copy the `README` contents in the `stories.mdx` file because it contains a link to `useResizeObserver`.  I can't change that link directly in the original `README` right now, as it would change the link in docz hosted Atlantis web. 
- _Stylelint Config_: For an unknown reason, auto-formatting on save was not working when trying to pull contents directly from the original `README`. This was stopping the file from loading in Storybook so chose to just copy original `README` into the `stories.mdx` file.

## Notes

Once we remove docz, this will be removed in cleanup
Screenshot 2023-02-24 at 2 19 39 PM

---

[In Atlantis we use Github's built in pull request reviews](https://help.github.com/en/articles/about-pull-request-reviews).

Random photo of Atlantis

### CODE REVIEW

**Review by rodrigoeidelvein — CHANGES_REQUESTED** [review]
The packages are showing the header from the MDX files
image

**Comment by taylorvnoj** [comment]
> The packages are showing the header from the MDX files image

@rodrigoeidelvein yup that will only remain until we remove docz - I worked with Darryl on this and it's ok for now since we can remove the header from the `mdx` files once docz is gone and then we publish the storybook version. I hope that makes sense, let me know if you have any questions

File: packages/hooks/readme.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+Shared hooks for components in Atlantis.
+
+## Hooks
+
+- [useResizeObserver](../?path=/docs/hooks-useresizeobserver--use-resize-observer)
+
+## Installing
+
+`npm install hooks @jobber/hooks`
```
Comment (rodrigoeidelvein) [inline packages/hooks/readme.stories.mdx:16]:
```suggestion
`npm install @jobber/hooks`
```

File: packages/stylelint-config/readme.stories.mdx [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import { Meta } from "@storybook/addon-docs";
+
+<Meta title="Packages/Stylelint Config" />
+
+# Stylelint Config
+
+## Installing
+
+`npm install stylelint @jobber/stylelint-config`
```
Comment (rodrigoeidelvein) [inline packages/stylelint-config/readme.stories.mdx:9]:
```suggestion
`npm install @jobber/stylelint-config`
```

**Review by rodrigoeidelvein — CHANGES_REQUESTED** [review]
Just some duplicates when installing the packages

**Review by rodrigoeidelvein — APPROVED** [review]
LGTM!

**Pull request merged**